eng/Dependencies.props

```diff
@@ -141,15 +141,8 @@ and are generated based on the last package release.
 
   <ItemGroup Label=".NET team dependencies (Non-source-build)" Condition="'$(DotNetBuildSourceOnly)' != 'true'">
     <LatestPackageReference Include="Microsoft.AspNet.WebApi.Client" />
-    <LatestPackageReference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.2.1" />
-    <LatestPackageReference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.3.1.x64" />
-    <LatestPackageReference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.3.1.x86" />
-    <LatestPackageReference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.5.0.x64" />
-    <LatestPackageReference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.5.0.x86" />
     <LatestPackageReference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.6.0.x64" />
```

Comment:
do we need 6.0? It's about to be out of support.

Reply:
It's still in support for a few months, I don't think we should remove it until it's officially out of support

Reply:
might be worth something on the backlog to do that else it will just show up later in s360

Reply:
This now falls under general 6.0 cleanup that I'll do once that's out of support, which I'm tracking w/ https://github.com/dotnet/aspnetcore-internal/issues/4591